src/Pages/Templates/TemplateDetails/components/TemplateActionDropdown.tsx

```diff
@@ -49,15 +49,23 @@ export default function TemplateActionDropdown() {
     }
   };
 
+  const hasRHELSubscription = subscriptions?.red_hat_enterprise_linux || false;
+  const isMissingRequirements = !rbac?.templateWrite || !hasRHELSubscription;
+  const missingRequirements: string = !rbac?.templateWrite
+    ? 'permission'
+    : !hasRHELSubscription
+      ? 'subscription (RHEL)'
+      : 'permission';
```

Comment:
Would the following not simplify this:
`rbac?.templateWrite && !hasRHELSubscription ? 'subscription (RHEL)' : 'permission'`

Reply:
resolved [here](https://github.com/content-services/content-sources-frontend/pull/328/commits/92ada247ce4e2cfe19fb4e6adac241f9ae08e54d) 🫡